app/stac_api/managers.py

```diff
@@ -153,6 +161,64 @@ def _parse_datetime_query(self, date_time):
             )
         return start, end
 
+    def filter_by_item_name(self, item_names_array):
+        '''Filter by item names parameter
+
+        Args:
+            item_names_array: list[string]
+                An array of ids (string)
+
+        Returns:
+            queryset filtered by a list of ids
+        '''
+        return self.filter(name__in=item_names_array)
+
+    def filter_by_collections(self, collections_array):
+        '''Filter by collections parameter
+
+        Args:
+            collections_array: list[string]
+                An array of collections (string)
+
+        Returns:
+            queryset filtered by a list of collections
+        '''
+        return self.filter(collection__name__in=collections_array)
+
+    def filter_by_intersects(self, intersects):
+        '''Filter by intersects parameter
+
+        Args:
+            intersects: string
+                Is a geojson formatted string
+
+        Returns:
+            queryset filtered by intersects
```

Comment:
You could add the `Raises:` in the docstring with the exception that might occur if the intersects is not correct.